### ISSUE TITLE
Can't set global export name programmatically ('name' option)

### ISSUE THREAD

**Issue by flawyte** [medikoo/modules-webmake#50]
Hi,

I'm trying to pass options to webmake to set the exposed variable name but it doesn't work :

```javascript
var webmake = require('webmake');

webmake('index.js', {
  name: 'SomeName', // Should export the module to 'window.SomeName' but doesn't
  output: 'dist/some-name.js'
});

});
```

Then when I include *dist/some-name.js* in an HTML page it says that `SomeName not defined`.

What am I doing wrong ?

**Comment by medikoo** [comment]
Thanks for reporting. For some reasons these option had only console support. I've fixed that. It's published as v0.3.37

**Comment by flawyte** [comment]
Thanks :+1:
